source/docs/software/advanced-controls/state-space/state-space-intro.rst

```diff
@@ -0,0 +1,198 @@
+
+.. image:: images/pendulum-closed-loop.png
+
+But with a real system, how can we choose an optimal :term:`gain` matrix K? While we can manually choose :term:`gains <gain>` and simulate the system response, or use tools like pole placement, modern control theory has a better answer: the Linear-Quadratic Regulator (LQR).
+
+The Linear-Quadratic Regulator
+------------------------------
+
+Linear-Quadratic Regulators works by finding a :term:`control law` that minimizes the following cost function, which weights the sum of :term:`error` and :term:`control effort` over time, subject to the linear :term:`system` dynamics :math:`\mathbf{\dot{x} = Ax + Bu}`.
+
+.. math::
+    J = \int\limits_0^\infty \left(\mathbf{x}^T\mathbf{Q}\mathbf{x} +
+    \mathbf{u}^T\mathbf{R}\mathbf{u}\right) dt
+
+The :term:`control law` that minimizes :math:`\mathbf{J}` can be written as :math:`\mathbf{u = K(r - x)}`, where :math:`r-x` is the :term:`error`.
+
+By adjusting the state excursion weight :math:`\mathbf{Q}` and :term:`control effort` weight :math:`\mathbf{R}`, the response of the system can be tuned to suit the application.
+
+.. note:: LQR design's :math:`\mathbf{Q}` and :math:`\mathbf{R}` matrices don't need discretization, but the :math:`\mathbf{K}` calculated for continuous-time and discrete time :term:`systems <system>` will be different.
+
+Bryson's Rule
+~~~~~~~~~~~~~
+
+Picking these :math:`\mathbf{Q}` and :math:`\mathbf{R}` weights can be done using Bryson's rule, which provides a simple form for these cost matrices. With Bryson's rule, the diagonals of the :math:`\mathbf{Q}` and :math:`\mathbf{R}` matrices are chosen based on the maximum acceptable value for each :term:`state` and :term:`input`. The nondiagonal elements are zero.
+
+.. math::
+    \begin{array}{cc}
+        \mathbf{Q} = \begin{bmatrix}
+            \frac{1}{x_{1,max}^2} & 0 & \ldots & 0 \\
+            0 & \frac{1}{x_{2,max}^2} & & \vdots \\
+            \vdots & & \ddots & 0 \\
+            0 & \ldots & 0 & \frac{1}{x_{n,max}^2}
+        \end{bmatrix} &
+        \mathbf{R} = \begin{bmatrix}
+            \frac{1}{u_{1,max}^2} & 0 & \ldots & 0 \\
+            0 & \frac{1}{u_{2,max}^2} & & \vdots \\
+            \vdots & & \ddots & 0 \\
+            0 & \ldots & 0 & \frac{1}{u_{n,max}^2}
+        \end{bmatrix}
+    \end{array}
+
+.. note::
+    Don't confuse Q and R with the elements we use to construct :math:`\mathbf{Q}` and :math:`\mathbf{R}` with using Bryson's rule! Q and R are matrices with dimensionality states by states and states by inputs respectively. We fill Q with as many "q elements" as the :term:`system` has :term:`states <state>`, and R with as may "r elements" as the :term:`system` has :term:`inputs <input>`.
```

Comment:
Q and R should either all be in boldface or none at all. Also R should be inputs x inputs.